assets/js/show-task.js

```diff
@@ -21,32 +22,50 @@ function showTask(inputValue, formattedDate, taskList, form, allTasks, id) {
   const date = document.createElement("p");
   date.innerHTML = "Created At: " + formattedDate;
 
+  const saveButton = document.createElement("button");
+  saveButton.innerText = "Save";
+  saveButton.classList.add("hide");
+
   const done = document.createElement("button");
-  done.classList.add("done");
   done.innerHTML = `<i class="fa-solid fa-check" id="done"></i>`;
   done.setAttribute("aria-label", "Mark task as done");
 
   const edit = document.createElement("button");
-  edit.classList.add("edit");
   edit.innerHTML = `<i class="fa-solid fa-pen" id="edit"></i>`;
   edit.setAttribute("aria-label", "Edit task");
 
   const dlt = document.createElement("button");
-  dlt.classList.add("delete");
   dlt.innerHTML = `<i class="fa-solid fa-trash-can" id="delete"></i>`;
   dlt.setAttribute("aria-label", "Delete task");
 
+  const revert = document.createElement("button");
+  revert.innerHTML = `<i class="fa-solid fa-trash-can" id="revert"></i>`;
+  revert.classList.add("hide");
+
   tools.appendChild(date);
+  tools.appendChild(saveButton);
   tools.appendChild(done);
   tools.appendChild(edit);
   tools.appendChild(dlt);
+  tools.appendChild(revert);
 
   task_content.appendChild(tools);
   task.appendChild(task_content);
   taskList.prepend(task);
 
   deleteTaskEvent(dlt, task, taskList, allTasks, id);
   doneTaskEvent(done, edit, inputElement, tools, allTasks, id);
+  editTaskEvent(
+    done,
+    edit,
+    dlt,
+    revert,
+    saveButton,
+    inputElement,
+    tools,
+    allTasks,
+    id
+  );
```

Comment:
Can we be more specific with the variable naming here? So that even the one who is not familiar with this project can still understand the meaning of each variable? 

For example: Here if `done` represents the element that the user clicks to mark a task as done, it may be written as `doneButton `or maybe `doneElement`.